components/coda/actions/list-docs/list-docs.mjs

```diff
@@ -0,0 +1,106 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda-list-docs",
+  name: "List Docs",
+  description: "Returns a list of docs accessible by the user. These are returned in the same order as on the docs page: reverse chronological by the latest event relevant to the user (last viewed, edited, or shared)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    coda,
+    docId: {
+      propDefinition: [
+        coda,
+        "docId",
+        (c) => c,
+      ],
+      description: "Show only docs copied from the specified doc ID",
+      optional: true,
+    },
+    workspaceId: {
+      type: "string",
+      label: "Workspace ID",
+      description: "Show only docs belonging to the given workspace",
+      optional: true,
+    },
+    folderId: {
+      propDefinition: [
+        coda,
+        "folderId",
+      ],
+      description: "Show only docs belonging to the given folder",
+    },
+    query: {
+      propDefinition: [
+        coda,
+        "query",
+      ],
+    },
+    isOwner: {
+      type: "boolean",
+      label: "Is Owner Docs",
+      description: "Show only docs owned by the user",
+      optional: true,
+    },
+    isPublished: {
+      type: "boolean",
+      label: "Is Published Docs",
+      description: "Show only published docs",
+      optional: true,
+    },
+    isStarred: {
+      type: "boolean",
+      label: "Is Starred Docs",
+      description: "If true, returns docs that are starred. If false, returns docs that are not starred",
+      optional: true,
+    },
+    inGallery: {
+      type: "boolean",
+      label: "In Gallery Docs",
+      description: "Show only docs visible within the gallery",
+      optional: true,
+    },
+    limit: {
+      propDefinition: [
+        coda,
+        "limit",
+      ],
+    },
```

Comment:
To differentiate the `limit` prop as it's used in this action (to limit the number of docs retrieved over multiple requests) from other actions, I'd suggest creating (and renaming to) a separate `max`/`maxItems` prop (e.g., [`twitter.app.js`](https://github.com/PipedreamHQ/pipedream/blob/9b9500f95e665a9599325b522b51a95334fd91c2/components/twitch/twitch.app.js#L26)).

Also, the `limit` prop's [`max`](https://github.com/PipedreamHQ/pipedream/blob/023b3cbafcabb126d149374449b8e62bbea660ba/components/coda/coda.app.mjs#L131) value, as defined in `coda.app.mjs`, is `50`. To support returning all items to the user, remove the `max` property from the `maxItems` prop, and make it optional.

Reply:
Addressed